geometry_msgs/Pose navigation_psoe
geometry_msgs/Pose face_psoe
bool mask
uint8 id